test/fixtures/azure-fixture/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.test.fixtures'
+
+description = 'Fixture for Azure external service'
+test.enabled = false
+
+dependencies {
+    compile project(':server')
+}
+
+task fixtureJar(type: Jar) {
+    dependsOn compileJava
+    from sourceSets.main.output
+}
+
+preProcessFixture {
+    dependsOn fixtureJar
+    doLast {
+        file("${testFixturesDir}/shared").mkdirs()
+        project.copy {
+            from project.tasks.fixtureJar.outputs.files
+            from project.configurations.compile
```

Comment:
This should technically be `from configurations.runtimeClasspath` since that's what we want. It's just in this particular case the compile and runtime classpaths are effectively the same.